packages/core/src/declareAtom.ts

```diff
@@ -121,8 +121,8 @@ export function declareAtom<TState>(
     depTree.fnsMap.forEach((_, key) => _tree.addFn(update, key))
   }
 
-  handler(_initAction, (state = initialState as TState) => state)
-  dependencyMatcher(handler)
+  on(_initAction, (state = initialState as TState) => state)
+  dependencyMatcher(on)
```

Comment:
Maybe keep handler? it is more descriptive

Reply:
`on` is friendly for new contributors